Makefile

```diff
@@ -117,23 +117,27 @@ run-test-mode: generate fmt-check lint manifests ## Run against the configured K
 	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev -test-mode -webhook-port=0
 
 .PHONY: install
-install: manifests ## Install CRDs into a cluster
+install: $(KUSTOMIZE) manifests ## Install CRDs into a cluster
 	$(KUSTOMIZE) build config/crd | kubectl apply -f -
```

Comment:
Could use `$<` in place of `$(KUSTOMIZE)` in all of these rules if we stop allowing people to replace it with their own binary.

Reply:
Is there an elegant way of specifying the second dependency? All I found is `$(word 2,$^)`, and it's ugly.

Reply:
Yeah, you're correct, there's nothing better than that. The only elegant solution would be to split that rule into two (or three).